### ISSUE TITLE
Download link fails with mixed content error

### ISSUE THREAD

**Issue by stdavis** [agrc/AddressPointEditor#106]
When attempting to click the download like, I get a mixed content error in the console and the download does not complete.

This is caused by ArcGIS Server returning a HTTP (not HTTPS) URL for the data. We need to add some code to switch out the `http` for `https` on the client.

**Comment by steveoh** [comment]
this is still an issue

**Comment by stdavis** [comment]
Yes, it appears to still be an issue. I wonder if this feature is just not ever used.

**Comment by steveoh** [comment]
It should be a link to open data to download the data.
